docs/sidebars.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       label: 'Rollout Features',
       items:[
         "rollout/rollout",
-        "rollout/appdeploy"
+        'rollout/appdeploy'
```

Comment:
revert?

Reply:
Hongchao said we should enable the format check.

Reply:
I will add fmt check in next PR